Gallery/TaylorDiagrams/NCL_taylor_6.py

```diff
@@ -0,0 +1,75 @@
+"""
+NCL_taylor_6.py
+===============
+This script illustrates the following concepts:
+   - Labelling a Taylor diagram
```

Comment:
I recommend mentioning that we heavily use GeoCAT-viz to make this plot and adding a link to the Taylor diagram documentation. It would be a great way to promote our other packages